grazer/src/main/java/edu/asu/diging/grazer/core/conceptpower/impl/ConceptpowerConcept.java

```diff
@@ -120,8 +147,13 @@ public void setType(ConceptpowerConceptType type) {
         this.type = type;
     }
     
-    public List<ConceptpowerAlternativeId> getAlternativeIds() {
-        return alternativeIds;
+    public List<String> getAlternativeIds() {
```

Comment:
It doesn't make much sense that setter and getter work with different datatypes. If you setter accepts ```ConceptpowerAlternativeId```s then your getter should return those.